server/graphql/types/src/types/asset_catalogue_item.rs

```diff
@@ -0,0 +1,84 @@
+use async_graphql::*;
+
+use graphql_core::simple_generic_errors::NodeError;
+use repository::assets::{
+    asset_catalogue_item::AssetCatalogueItem, asset_catalogue_item_row::AssetCatalogueItemRow,
+};
+use service::ListResult;
+
+#[derive(PartialEq, Debug)]
+pub struct AssetCatalogueItemNode {
+    pub asset_catalogue_item: AssetCatalogueItem,
+}
+
+#[derive(SimpleObject)]
+pub struct AssetCatalogueItemConnector {
+    total_count: u32,
+    nodes: Vec<AssetCatalogueItemNode>,
+}
+
+#[Object]
+impl AssetCatalogueItemNode {
+    pub async fn id(&self) -> &str {
+        &self.row().id
+    }
+
+    pub async fn asset_category_id(&self) -> &str {
+        &self.row().category_id
+    }
+
+    pub async fn asset_class_id(&self) -> &str {
+        &self.row().class_id
+    }
+
+    pub async fn code(&self) -> &str {
+        &self.row().code
+    }
+    pub async fn manufacturer(&self) -> Option<String> {
+        self.row().manufacturer.as_ref().map(|it| it.clone())
+    }
+    pub async fn model(&self) -> &str {
+        &self.row().model
+    }
+    pub async fn asset_type_id(&self) -> &str {
```

Comment:
Might be nice to have this next to the other `asset_` fields? I thought it was missing at first...